framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -170,6 +170,15 @@
         
         String queryFileExtension = modeler.matrices.get(0).inputFile;
         String expectedFileExtension = modeler.matrices.get(0).expectedFile;
+        String tempQueryExt = queryFileExtension;
+        if(TestDriver.cmdParam.negativeCasesInclusion == true){
+            LOG.info(LINE_BREAK);
+            LOG.info("fail added");
+            LOG.info(LINE_BREAK);
+            queryFileExtension += "|"+tempQueryExt+".fail"+"|"+tempQueryExt+".failing";
```

Comment:
Use static final variables for literals. We could also look this up in the properties file.